var/spack/repos/builtin/packages/net-snmp/package.py

```diff
@@ -20,3 +21,6 @@ class NetSnmp(AutotoolsPackage):
     def configure_args(self):
         args = ['--with-defaults', 'LIBS=-ltinfo']
         return args
+
+    def install(self, spec, prefix):
+        make('install', parallel=False)
```

Comment:
Read your comment. This is perfect to have only `make install` done serially.

Reply:
Great, thanks for the feedback!